internal/bminventory/inventory.go

```diff
@@ -104,6 +104,7 @@ type Config struct {
 	DefaultServiceNetworkCidr       string            `envconfig:"SERVICE_NETWORK_CIDR" default:"172.30.0.0/16"`
 	WithAMSSubscriptions            bool              `envconfig:"WITH_AMS_SUBSCRIPTIONS" default:"false"`
 	ISOImageType                    string            `envconfig:"ISO_IMAGE_TYPE" default:"full-iso"`
+	DisableIPv6                     bool              `envconfig:"DISABLE_IPV6" default:"false"`
```

Comment:
Is true the right default?.  People complained about seeing IPv6 networks.

Reply:
I don't know if it's correct to assume that people who have IPv6 addresses in their environments intend to use them. In which case it would be annoying to need to turn the IPv6 support explicitly. Moreover, I don't have a way to verify this assumption right now. So in the meantime I disabled IPv6 in the cloud (because it can't work) and enabled by default in self-deployed environments such as on-prem. If that turns out not good, we'll change the default to "true".

Reply:
Please verify.  @avishayt what do you think?

Reply:
It's up to @nirfarkas and Moran to decide